src/servers/sapphire_zone/Zone/Land.cpp

```diff
@@ -152,6 +152,13 @@ uint16_t Core::Land::getZoneId() const
   return m_zoneId;
 }
 
+Core::HousePtr Core::Land::getHouse() const
+{
+  if( !m_house )
```

Comment:
should rename m_house to m_pHouse to indicate it being a ptr.
can also just directly return m_pHouse as that will be nullptr if not set.